workspaces/jupyterlab-python/requirements.txt

```diff
@@ -0,0 +1,25 @@
+automata-lib==7.1.0
+beautifulsoup4==4.12.2
+bokeh==3.2.2
+colormath==3.0.0
+defusedxml==0.7.1
+folium==0.14.0
+ipython==8.14.0
+matplotlib==3.7.2
+nbconvert==7.8.0
+nbformat==5.9.2
+networkx==3.1
+nltk==3.8.1
+numpy==1.25.2
+openpyxl==3.1.2
+pandas==2.0.3
+Pillow==10.0.0
+PuLP==2.7.0
+Pygments==2.16.1
+pygraphviz==1.11
```

Comment:
This line is the only difference between this file and the one in `graders/python/requirements.txt`. I couldn't get version 1.6 to install successfully. We should really look at upgrading the Python grader image so that it can support the latest version of Pygraphviz. Right now we're stuck on Amazon Linux 2, which won't be seeing any more changes.

Reply:
Yes, it would be great to use something fully up to date here.